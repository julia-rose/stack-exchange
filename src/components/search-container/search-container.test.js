import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import axios from 'axios';
import SearchContainer from './index';

describe('Search Container', () => {
  const mockSetSearchTerm = jest.fn();
  const component = shallow(
    <SearchContainer setNewSearchTerm={mockSetSearchTerm} />
  );

  it('renders and matches snapshot', () => {
    const component = renderer.create(
      <SearchContainer />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});