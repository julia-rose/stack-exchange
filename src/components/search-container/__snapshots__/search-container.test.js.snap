// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Container renders and matches snapshot 1`] = `
<div
  className="SearchContainer"
>
  <input
    onChange={[Function]}
    type="text"
    value=""
  />
  <button
    onClick={[Function]}
  >
    Search
  </button>
</div>
`;
